### ISSUE TITLE
eth: Initiating 8 swaps fails on simnet

### ISSUE THREAD

**Issue by martonp** [decred/dcrdex#1304]
When trying to initiate greater than 7 swaps in the simnet, the transaction will fail. The contract is fine as loaded the contract into remix and initiated 100, so the simnet is somehow setup incorrectly.

**Comment by JoeGruffins** [comment]
I'll look into this if it's still a problem.

**Comment by martonp** [comment]
Yeah it is, easy way to see it is in the `TestRedeemGas` where 7 swaps are initiated, just change it to 8.

**Comment by JoeGruffins** [comment]
It actually seems to happen at nine initiations now with recent improvements to the efficiency.

The error is:`nodeclient_harness_test.go:706: Unable to initiate swap: tx fee (1.04 ether) exceeds the configured cap (1.00 ether)`

It seems this is due a a hard limit on fee+value. It can be seen as a configurable value here https://geth.ethereum.org/docs/interface/command-line-options `--rpc.txfeecap value                Sets a cap on transaction fee (in ether) that can be sent via the RPC APIs (0 = no cap) (default: 1)`

Do we just want to set to zero? 

Actually not sure where/how to set these RPC values. They don't seem to be accessible through our internal node's config. Maybe just haven't found them yet.

**Comment by buck54321** [comment]
We adopt a bunch of defaults [here](https://github.com/decred/dcrdex/blob/4ee4c478fe7941bff575af1c39617d7945ee091e/client/asset/eth/node.go#L207), including [this one](https://github.com/ethereum/go-ethereum/blob/619a3e70858e60240ce1df75bdf65ba748387e57/eth/ethconfig/config.go#L95). Just need to `ethCfg.RPCTxFeeCap = 0`, I guess.

**Comment by JoeGruffins** [comment]
The test uses a fee of 1000gwei. The math checks out gas used is 135000 + (113000 * 8) or 1039000 times fee gives you 1.039 ether. Normally fee wouldn't be that high.

`ethCfg.RPCTxFeeCap = 0` works. I was looking at the wrong config, where we create the node.

I think setting this is fine if we are monitoring the gas fee anyway and respect the user's max fee rate.

**Comment by JoeGruffins** [comment]
It looks like we currently don't enforce the user's max fee prefs, so need to do that.
